packages/eslint-plugin/tests/rules/no-type-alias.test.ts

```diff
@@ -131,66 +131,6 @@ ruleTester.run('no-type-alias', rule, {
       code: 'type Foo = 1 | (2 & 3);',
       options: [{ allowAliases: 'in-unions-and-intersections' }],
     },
-    {
-      code: 'type Foo = `a-${number}`;',
-      options: [{ allowAliases: 'always' }],
-    },
-    {
-      code: 'type Foo = `a-${number}` | `b-${number}`;',
-      options: [{ allowAliases: 'always' }],
-    },
-    {
-      code: 'type Foo = `a-${number}` | `b-${number}`;',
-      options: [{ allowAliases: 'in-unions-and-intersections' }],
-    },
-    {
-      code: 'type Foo = `a-${number}` | `b-${number}`;',
-      options: [{ allowAliases: 'in-unions' }],
-    },
-    {
-      code: 'type Foo = `a-${number}` | `b-${number}` | `c-${number}`;',
-      options: [{ allowAliases: 'always' }],
-    },
-    {
-      code: 'type Foo = `a-${number}` | `b-${number}` | `c-${number}`;',
-      options: [{ allowAliases: 'in-unions-and-intersections' }],
-    },
-    {
-      code: 'type Foo = `a-${number}` | `b-${number}` | `c-${number}`;',
-      options: [{ allowAliases: 'in-unions' }],
-    },
-    {
-      code: 'type Foo = `a-${number}` & `b-${number}`;',
-      options: [{ allowAliases: 'always' }],
-    },
-    {
-      code: 'type Foo = `a-${number}` & `b-${number}`;',
-      options: [{ allowAliases: 'in-unions-and-intersections' }],
-    },
-    {
-      code: 'type Foo = `a-${number}` & `b-${number}`;',
-      options: [{ allowAliases: 'in-intersections' }],
-    },
-    {
-      code: 'type Foo = `a-${number}` & `b-${number}` & `c-${number}`;',
-      options: [{ allowAliases: 'always' }],
-    },
-    {
-      code: 'type Foo = `a-${number}` & `b-${number}` & `c-${number}`;',
-      options: [{ allowAliases: 'in-unions-and-intersections' }],
-    },
-    {
-      code: 'type Foo = `a-${number}` & `b-${number}` & `c-${number}`;',
-      options: [{ allowAliases: 'in-intersections' }],
-    },
-    {
-      code: 'type Foo = `a-${number}` | (`b-${number}` & `c-${number}`);',
-      options: [{ allowAliases: 'always' }],
-    },
-    {
-      code: 'type Foo = `a-${number}` | (`b-${number}` & `c-${number}`);',
-      options: [{ allowAliases: 'in-unions-and-intersections' }],
-    },
```

Comment:
Could you add these test cases back in, please? This shouldn't be a breaking change PR. They should still be valid code.